homeassistant/components/wiz/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Config flow for WiZ Platform."""
+import logging
+
+from pywizlight import wizlight
+from pywizlight.exceptions import WizLightConnectionError, WizLightTimeOutError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
```

Comment:
I'd remove the name from the form and use the host as title. The user can rename entry title and also devices and entities since we set unique ids for the entities.

Reply:
https://github.com/home-assistant/core/pull/65746